src/pages/patientView/mutation/column/ProteinChangeColumnFormatter.spec.jsx

```diff
@@ -1,21 +1,72 @@
-import * as ProteinChangeColumnFormatter from './ProteinChangeColumnFormatter';
+import ProteinChangeColumnFormatter from './ProteinChangeColumnFormatter';
+import styles from './style/proteinChange.module.scss';
 import React from 'react';
 import { assert } from 'chai';
 import { shallow, mount } from 'enzyme';
 import sinon from 'sinon';
 
-describe('ProteinChangeColumnFormatter', () => {
+describe('ProteinChangeColumnFormatter (customized for patient view)', () => {
+    const germlineMutation = {
+        proteinChange: "Q616K",
+        mutationStatus: "Germline"
+    };
 
-    before(()=>{
+    const somaticMutation = {
+        proteinChange: "Q616L",
+        mutationStatus: "Somatic"
+    };
 
-    });
+    const tableData = [[germlineMutation], [somaticMutation]];
 
-    after(()=>{
+    let germlineComponent, somaticComponent;
+
+    before(() => {
+
+        let data = {
```

Comment:
`data` is accessed only in `before`, so I didn't want to define it in the body. In the body, I prefer to define only the variables accessed by `it`.